src/CoreShop/Bundle/CoreBundle/Migrations/Version20191007174441.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace CoreShop\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Pimcore\Migrations\Migration\AbstractPimcoreMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20191007174441 extends AbstractPimcoreMigration
+{
+    /**
+     * @param Schema $schema
+     *
+     * @throws \Doctrine\DBAL\Schema\SchemaException
+     */
+    public function up(Schema $schema)
+    {
+
+        if ($schema->hasTable('coreshop_payment_provider')) {
+
+            $table = $schema->getTable('coreshop_payment_provider');
+
+            if ($table->hasColumn('logo')) {
+                $table->dropColumn('logo');
+            }
+
+            if ($table->hasColumn('logoId')) {
```

Comment:
not needed. Why even change the table anyway? we had this migration already didn't we?